src/main/java/frc/robot/RobotContainer.java

```diff
@@ -140,24 +137,8 @@ else if (selectorOne == 1 || selectorTwo == 1) {
     }
 
     public void joystickRumble(double leftValue, double rightValue) {
-        //Joystick rumble (driver feedback). leftValue/rightValue sets vibration force.
+        //Joystick rumble (driver feedback). leftValue/rightValue sets vibration force (0.0-1.0).
         driverStick.setRumble(RumbleType.kLeftRumble, leftValue);
         driverStick.setRumble(RumbleType.kRightRumble, rightValue);
     }
-
-    /**
-     * Initialize the current RobotContainer instance
-     */
-    public static void init() {
-        if (currentInstance == null) {
-            currentInstance = new RobotContainer();
-        }
-    }
-
-    public static RobotContainer getInstance() {
```

Comment:
This code that I added in [pull request #124](https://github.com/FRC2706/2020-2706-Robot-Code/pull/124) was not needed. RobotContainer is public.